packages/twenty-front/src/modules/object-record/hooks/useDeleteOneRecordMutation.ts

```diff
@@ -26,12 +27,11 @@ export const useDeleteOneRecordMutation = ({
   );
 
   const deleteOneRecordMutation = gql`
-    mutation DeleteOne${capitalizedObjectName}($idToDelete: ID!)  {
-      ${mutationResponseField}(id: $idToDelete) {
-        id
-      }
-    }
-  `;
+  mutation DeleteOne${capitalizedObjectName}($idToDelete: ID!) {
+    ${mutationResponseField}(id: $idToDelete)
+    ${mapSoftDeleteFieldsToGraphQLQuery(objectMetadataItem)}
```

Comment:
where do you use the answer of this query? I thought it would have been used to update the cache but it isn't, is it?

Reply:
Ah! saw your change, good point :)